src/Controller/Ticket/TicketController.php

```diff
@@ -43,4 +44,13 @@ public function add(
 
         return $this->success([], Response::HTTP_CREATED);
     }
+
+    #[Route('/tickets/cancel/{id}', name: 'cancel', methods: 'POST')]
+    public function cancel(int $id, TicketRepository $ticketRepository, TicketService $ticketService)
+    {
+        $ticket = $ticketRepository->find($id);
+        $ticketService->cancel($ticket);
+
+        return $this->success([]);
```

Comment:
Where is status code when it success?

Reply:
status code is auto set to 200 in success function